dist/autofill.js

```diff
@@ -744,32 +744,33 @@ class Form {
     const handler = e => {
       if (this.tooltip) return; // Checks for mousedown event
 
-      if (e.type === 'mousedown') {
+      if (e.type === 'pointerdown') {
         if (!e.isTrusted) return;
         const isMainMouseButton = e.button === 0;
         if (!isMainMouseButton) return;
       }
 
       if (this.shouldOpenTooltip(e, e.target)) {
-        if (isEventWithinDax(e, e.target) || isDDGApp && !isApp) {
+        if (isEventWithinDax(e, e.target) || isMobileApp) {
           e.preventDefault();
           e.stopImmediatePropagation();
         }
 
         this.touched.add(e.target);
         this.attachTooltip(this, e.target);
       }
-    }; // TODO: on mobile, focus could open keyboard before tooltip
-
+    };
 
-    ['mousedown', 'focus'].forEach(ev => this.addListener(input, ev, handler, true));
+    const events = ['pointerdown'];
+    if (!isMobileApp) events.push('focus');
```

Comment:
How come mobile doesn't have a focus handler?

Reply:
It does. What I mean is that you can't reliably cancel the focus event before the keyboard opens, so it ended up in a weird experience where either the keyboard would flash before our native component or it would appear after interactive with the component.

Reply:
Ah you answered elsewhere 👍🏻